src/components/deckData/deckCard.js

```diff
@@ -0,0 +1,105 @@
+import React, {useState, useEffect} from "react";
+import {Card, Avatar, Modal, Button, Form, Input} from "antd";
+import {getAllDecks, deleteDeck, getDeckById} from "../../state/actions/decks";
+import {EditOutlined, DeleteOutlined, PlayCircleOutlined} from "@ant-design/icons";
```

Comment:
I think it's not a question for you, Nick, I have asked Babatunde already why he installed anotherr version of antd icons lib on top of what we have.

Reply:
Not my component